roles/kubernetes/preinstall/tasks/pre_upgrade.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: "Pre-upgrade | check if old credential dir exists"
+  stat:
+    path: "{{ inventory_dir }}/../credentials"
```

Comment:
We need to drop the `{{ inventory_dir}}/../` part of this check for the old credentials since they were previously just `credentials`.

Reply:
I don't trust it without an absolute path reference

Reply:
the old credentials were generated under `kubespray/credentials`. So if your inventory dir is not at the same directory level as kubespray, this check will be invalid.

Reply:
oh I catch your drift

Reply:
This is failing because of python interpreter path for centos